### DISCUSSION TITLE
Error when opening multiple channels:

### DISCUSSION THREAD

**Discussion by BonzoDogDooDah** [lightningnetwork/lnd#7931] [Troubleshooting]
Hello all,

We have encounterd some errors when attempting to open a few channels. I have attached two images below, one showing the funds we have on hand and the other showing the error message we recieved. It appears to us that the error is saying we do not have enough funds to open the channel but our balance says otherwise? We tried  to open a few other channels to different nodes with each amount being 1.5 million sats each, which appears to meet each nodes prerequisites so not sure why the errors are popping up?

Please let us know if you have any ideas.

Image links are below.

https://ibb.co/jJdVxJ4

https://ibb.co/0sYM3fC

Thank you for your help.

Orange_ Bonzo

**Comment by guggero** [comment]
Not sure what the symbols of these screenshots mean. An output of `lncli walletbalance` would be appreciated. Otherwise I can just guess that there aren't enough _confirmed_ funds available.

**Comment by BonzoDogDooDah** [comment]
Hi Oliver,

Wow thank you for the quick reply. 

Sorry using Umbrel but I will soon be where the node is and try the command app and type in: lncli walletbalance and see how that goes.

I thought that when the Lightning icon went from yellow to purple and there were no unconfirmed messages in that same area of Thunderhub that we were good to go but I suppose the funds were probably just not confirmed yet, as you mentioned below.

Thanks again for your support.

Orange_Bonzo

-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



28 Aug 2023, 20:56 by ***@***.***:
